src/components/Textarea/Textarea.scss

```diff
@@ -0,0 +1,74 @@
+@use '../../utils/index';
+
+$height-size: 50px;
+$width-size: 40px;
+
+.moonstone-textarea-container {
+    position: relative;
+
+    display: inline-flex;
+    align-items: center;
+    width: 100%;
+    min-width: $width-size;
+    min-height: $height-size;
+
+    color: var(--color-gray_dark);
+
+    cursor: text;
+}
+
+.moonstone-textarea {
+    @extend %variant-caption;
+
+    display: block;
+    width: 100%; // CSS trick to force element to shrink
+    min-width: 50px;
+    min-height: $height-size;
+    margin: 0; // Reset for Safari
+
+    border: var(--border-selector);
+    border-radius: var(--radius_small);
+    outline: none;
+    resize: none;
+
+    -webkit-tap-highlight-color: transparent;
```

Comment:
You don't need to add prefix like `-webkit-`, we normally have with autoprefixer.